src/helpers.js

```diff
@@ -49,9 +47,9 @@ const getConfigsFromProcess = function (defaults) {
   var result = {}
 
   // Walk the defaults object and map the names back to process.env names so we can find them
-  _.forEach(defaults, (val, config) => {
+  Object.keys(defaults).forEach( (config) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [There should be no space after this paren.](https://app.codacy.com/manual/mailfrom/aem-packager/pullRequest?prid=5238672)